units/systemd-resolved-varlink.socket

```diff
@@ -0,0 +1,21 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=io.systemd.Resolve Socket
```

Comment:
Please call this "Resolve Service Varlink Socket", as we don't want impl details in description

This should carry

```
DefaultDependencies=no
Before=sockets.target
```

following resolved.service